matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/call/DefaultCallSignalingService.kt

```diff
@@ -58,18 +58,32 @@ internal class DefaultCallSignalingService @Inject constructor(
 
     private val activeCalls = mutableListOf<MxCall>()
 
-    private var cachedTurnServerResponse: TurnServerResponse? = null
+    private var cachedTurnServerResponse = object {
+
+        private val MIN_TTL = 60
+
+        private val now = { System.currentTimeMillis() / 1000 }
```

Comment:
This is a detail, but I more used to write:
`private fun now() = System.currentTimeMillis() / 1000`
Maybe there is an advantage I do not know to use `val`?